### DISCUSSION TITLE
请问Web端的音乐播放是怎么做的呢

### DISCUSSION THREAD

**Discussion by WangYueYang** [qier222/YesPlayMusic#1396] [Q&A]
# 请问Web端的音乐播放是怎么做的呢

我找到了 utils/Player.js,看了下代码有些复杂没有看太懂，我在页面上也没有找到和音频播放相关的HTML标签。想问下是如何实现播放音乐的呢。求解答

**Comment by pan93412** [comment]
底層是 Howler

**Reply by WangYueYang** [reply]
特别感谢您的及时回复，我去了解一下
